spec/memoizable/dependency_spec.rb

```diff
@@ -291,6 +292,17 @@ def calc
       }.to change { record.calc_count }.by(1)
     end
 
+    it 'locally caches computation to extract dependencies for an arel query' do
+      record = SpecModel.create!(a: 1)
+      record.calc_count = 0
+      RedisMemo::Cache.with_local_cache do
+        expect(RedisMemo::MemoizeMethod).to receive(:extract_dependencies).twice.and_call_original
```

Comment:
Why is the called twice? 🤔

Reply:
It's called once for extracting the dependencies of the calc method, and then the arel query it depends on. Though when the method is actually called and the Arel query is executed, extracting the dependnency of the Arel query isn't done again.

I can also try to be more specific about the arguments with which extract_dependencies is called